src/modelfree/score_agent.py

```diff
@@ -75,6 +114,15 @@ def default_score_config():
 @score_ex.main
 def score_agent(_run, _seed, env_name, agent_a_path, agent_b_path, agent_a_type, agent_b_type,
                 record_traj, record_traj_params, num_env, episodes, render, videos, video_dir):
+    if videos:
+        if video_dir is None:
+            logging.info("""No directory provided for saving videos; using a tmpdir instead,
```

Comment:
For just two-lines I'd probably use `"first line" "second line"` instead of triple quotes, but no strong preference.